pkg/havener/locations.go

```diff
@@ -206,16 +206,19 @@ func PathToHelmChart(input string) (string, error) {
 	}
 
 	// Each switch case applies an specific regular expression for an expected input.
-	switch {
+	switch urlRegex := regexp.MustCompile(`^((.*)@)?((https://|http://).+)$`); {
```

Comment:
any reasons why the previous regexp `^(.+)[@](.+)$` is removed(see https://regex101.com/r/lB7oO0/2), it gives you 2 groups for the match, that can be use directly. The new regex, gives you 4 groups, and you need to select fgroup 2 and 3

Reply:
https://regex101.com/r/lB7oO0/1/tests Provides some test cases for this regex. The difference to the other one is that here you can omit the relative path. If no path is provided, it will use the root directory of the .zip file. (Example with old regex: https://regex101.com/r/lTJTxY/2)

Reply:
thanks, now its clear to me.